src/Gui/MainWindow.cpp

```diff
@@ -1665,6 +1666,7 @@ void MainWindow::_updateActions()
         Application::Instance->commandManager().testActive();
     }
     d->actionUpdateDelay = 0;
+    getMainWindow()->setWindowTitle(activeWindow()->buildWindowTitle());
```

Comment:
What if there is no active window at all? Seems like the reason for the crash.

Reply:
to get the window modified asterisk